test/Rules.js

```diff
@@ -1,252 +1,336 @@
 var assert = require("assert");
 var Rules = require("../lib/Rules");
 
+describe("Rules", function () {
```

Comment:
Since this project's 2-spaced instead of 4, would you be up for changing this to 2-spaced indents and re-pushing, so I can tell what's changed?

I really should ad an .editorconfig in this project. :frowning: